data/data.js

```diff
@@ -1,73 +1,73 @@
-    "ps1": ["2006", "Powershellは、2006年にMicrosoftによって開発されたプログラミング言語である。", "Powershell"],
-    "py": ["1991", "Pythonは、1991年にGuido van Rossumによって開発されたプログラミング言語である。", "Python"],
-    "raku": ["2000", "Rakuは、2000年にLarry Wallによって開発されたプログラミング言語である。", "Raku"],
-    "rb": ["1995", "Rubyは、1995年にYukihiro Matsumotoによって開発されたプログラミング言語である。", "Ruby"],
-    "rdr": ["2018", "RDRは、2018年にRDR Groupによって開発されたプログラミング言語である。", "RDR"],
-    "re": ["2015", "Reasonは、2015年にFacebookによって開発されたプログラミング言語である。", "Reason"],
-    "res": ["2018", "Rescriptは、2018年にJordan Walkeによって開発されたプログラミング言語である。", "Rescript"],
-    "rack": ["1995", "Racketは、1995年にMatthew Flattによって開発されたプログラミング言語である。", "Racket"],
-    "rs": ["2010", "Rustは、2010年にMozilla Researchによって開発されたプログラミング言語である。", "Rust"],
-    "sb3": ["2003", "Scratchは、2003年にMIT Media Labによって開発されたプログラミング言語である。", "Scratch"],
-    "scala": ["2003", "Scalaは、2003年にMartin Oderskyによって開発されたプログラミング言語である。", "Scala"],
-    "scm": ["1975", "Schemeは、1975年にGerald Jay Sussmanによって開発されたプログラミング言語である。", "Scheme"],
-    "svelte": ["2016", "Svelteは、2016年にRich Harrisによって開発されたプログラミング言語である。", "Svelte"],
-    "swift": ["2014", "Swiftは、2014年にAppleによって開発されたプログラミング言語である。", "Swift"],
-    "tex": ["1978", "TeXは、1978年にDonald Knuthによって開発されたプログラミング言語である。", "TeX"],
-    "ts": ["2012", "TypeScriptは、2012年にMicrosoftによって開発されたプログラミング言語である。", "TypeScript"],
-    "v": ["2019", "Vは、2019年にAlexander Medvednikovによって開発されたプログラミング言語である。", "V"],
-    "vb": ["1996", "VBScriptは、1996年にMicrosoftによって開発されたプログラミング言語である。", "VBScript"],
-    "vim": ["1991", "Vim scriptは、1991年にBram Moolenaarによって開発されたプログラミング言語である。", "Vim script"],
-    "vue": ["2014", "Vue.jsは、2014年にEvan Youによって開発されたプログラミング言語である。", "Vue.js"],
-    "wat": ["2018", "watは、2018年にWebAssembly Community Groupによって開発されたプログラミング言語である。", "wat"],
-    "wy": ["2018", "WenyanLangは、2018年にYiyi Wangによって開発されたプログラミング言語である。", "WenyanLang"],
-    "zen": ["2015", "ZenScriptは、2015年にCraftTweakerによって開発されたプログラミング言語である。", "ZenScript"],
-    "zig": ["2015", "Zigは、2015年にAndrew Kelleyによって開発されたプログラミング言語である。", "Zig"],
+    "adb": ["2003", "Androidデバイスと通信するためのコマンドラインツール。デバッグや他の開発目的に利用される", "Android Debug Bridge"],
+    "appleScript": ["1993", " AppleのMac OSで使用される自動化スクリプト言語", "AppleScript"],
+    "async.py": ["2015", "Pythonの非同期プログラミングの一部。非同期I/Oを効率的に行うために使用される", "Async Python"],
+    "bash": ["1989", "Unixシェルとコマンド言語。Unix系OSで広く使用されている。", "Bash"],
+    "bat": ["1989", "Windowsのバッチファイルスクリプト。主に自動化タスクに使用される", "Bat"],
+    "c": ["1972", " 汎用的なプログラミング言語で、OSや組み込みシステムの開発によく使用される", "C"],
+    "clj": ["2007", "Lispに基づいた関数型プログラミング言語で、JVM上で動作する", "Clojure"],
+    "coffee": ["2009", "JavaScriptをより簡潔に書くための言語。JavaScriptにコンパイルされる", "CoffeeScript"],
+    "cpp": ["1983", "C言語にオブジェクト指向プログラミングとジェネリックプログラミングを追加した言語", "C++"],
+    "cr": ["2014", "高性能で型安全なRuby風の言語", "crystal"],
+    "d": ["2001", "C++の代替として設計されたオブジェクト指向プログラミング言語", "D"],
+    "dart": ["2011", "Googleが開発した、クライアント開発に特化した言語。Flutterフレームワークと一緒に使われることが多い", "Dart"],
+    "dash": ["2010", "Unixシェルの一種。シェルスクリプトの実行に特化している", "Dash"],
```

Comment:
参考: https://en.wikipedia.org/wiki/Almquist_shell#dash

```suggestion
    "dash": ["2010", "Unixシェルの一種。実装がスリムで実行速度が速い", "Dash"],
```